catalog-info.yaml

```diff
@@ -57,3 +57,34 @@ spec:
           access_level: MANAGE_BUILD_AND_READ
         everyone:
           access_level: READ_ONLY
+
+# Declare test execution on PR and merge to main
+---
+apiVersion: backstage.io/v1alpha1
+kind: Resource
+metadata:
+  name: buildkite-pr-main-mr-test
+  description: PR and Merge-to-Main Test
+  links:
+    - title: Pipeline
+      url: https://buildkite.com/elastic/pr-main-mr-test
+
+spec:
+  type: buildkite-pipeline
+  owner: group:docs
+  system: buildkite
+  implementation:
+    apiVersion: buildkite.elastic.dev/v1
+    kind: Pipeline
+    metadata:
+      name: pr-main-mr-test
+    spec:
+      repository: elastic/docs
+      pipeline_file: ".buildkite/pr_main_mr_test.yml"
+      provider_settings:
+        trigger_mode: none
```

Comment:
is this on purpose while developing the pipeline only, or a carry over ?

Reply:
then maybe remove it so PRs and merges to main trigger builds.
You may want to set build_branches as false - see https://docs.elastic.dev/ci/getting-started-with-buildkite-at-elastic#2-adjust-your-pipeline-settings

Reply:
I have added param "branch_configuration: "main"" to pipeline spec and removed the provider settings, since the default trigger is "code". As far as I understood "build_branches: false" will affect main branch as well, and it's not what we want in this case.